pipelines.adoc

```diff
@@ -159,23 +159,57 @@ A sample `manual-pipeline-run-template.yaml` file is provided in the link:https:
 oc apply -f <collection-name>-pipeline-run.yaml
 ```
 
-== Checking the status of the pipeline run
+== Execute pipelines manually from the command line for your custom built Collections
 
-You can check the status of the pipeline run from the OKD console,
-command line, or Tekton dashboard.
+If you have your own custom build collection image and want to run any pipelines against it instead of out of box kabanero collections, then you need to follow below steps as per the required scenarios.
 
+=== Setup of container registry url for the custom Collection image
 
-===  Ashish Execute pipelines manually from the command line for your custom built Collections
+By default the pipelines assume you are pulling the collection from `kabanero` so it looks for the collection image in dockerhub repository. If you want to use your container registry URL where you have the collection image you would need to follow below steps
 
-If you have your own custom build collection image and want to run any pipelines against it instead of out of box kabanero collections, then you need to follow below steps as per the required scenarios.
+. After you have cloned kabanero-pipelines repository find the configmap template file `collection-image-registry-map.yaml` and add your container registry URL in it for the key `default-collection-image-registry-url` 
+. asdasd
 
-=== Setup of container registry url for the custom Collection image
 
-. By default the pipelines assume you are pulling the collection from `kabanero` so it looks for the collection image in dockerhub repository. If you want to use
+ cd kabanero-pipelines/pipelines/common/
+ vi collection-image-registry-map.yaml
+
+. Apply the above configmap file which will set your container registry.
+
+ oc apply -f collection-image-registry-map.yaml
+
+==== For Collection image stored in container registry with internal route url on the OCP cluster
+
+For internal openshift registry you need to only setup above `collection-image-registry-map.yaml` with the internal registry url.
```

Comment:
Also maybe add that the service account should let the pipelines pull from the registry without needing to configure a secret.